app/helpers/db.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+import psycopg2 as psy
+
+from flask import g
+
+from app import cache
+from app import settings
+
+logger = logging.getLogger(__name__)
+
+
+@cache.cached(key_prefix='get_topics_from_db')
+def get_topics():
+    '''Get a list with all topics from bod
+
+    Returns:
+        A List with the topics
+    '''
+    try:
+        db_connection = get_db_connection()
+        with db_connection.cursor() as cursor:
+            # select records from DB
+            cursor.execute("""
+                SELECT topic FROM "re3".topics
+                """)
+            total_records = cursor.rowcount
+            logger.debug("Found %s records", total_records)
+            # iterate through table
+            _topics = ['all'] + [record[0] for record in cursor]
+    except psy.Error as error:
+        logger.exception('Failed to retrieve topics: %s', error)
+        _topics = settings.FALLBACK_TOPICS
+        logger.error("Connection to the db could not be established." \
+        " Using fallback %s", _topics)
+    logger.debug(
+        "Was not cached yet: get_topics_from_db: List of topics has been generated %s.", _topics
+    )
+    return _topics
+
+
+@cache.memoize()
+def get_translation(msg_id, lang):
+    '''Get translation from bod table translations
+
+    Args:
+        msgi_id:
+            String with the key to be translated
+        lang:
+            The lang to which will have to be translated
+
+    Returns:
+        String translated string or original string, if not in db
+
+    '''
+    # sanitize lang
+    lang = lang if lang in ['de', 'fr', 'it', 'rm', 'en'] else 'de'
+    query = f"SELECT {lang} FROM translations WHERE msg_id like '{msg_id}%'"
+    logger.debug('function get_translation: not cached, with query %s', query)
+    try:
+        db_connection = get_db_connection()
+        with db_connection.cursor() as cursor:
+            cursor.execute(query)
+            result = cursor.fetchone()
+            # if msg_id not in table translations
+            if result is not None:
+                translation = result[0]
+            else:
+                translation = msg_id
```

Comment:
In this case an info or even a warning message would be good, this would allow use through logging to find out which translation are missing. e.g.
```python
logger.warning('Missing translation for %s', msg_id, extra={"missing_translation": True})
```
Note the `extra` allow to easily filter those message in kibana.